task08_rivka_adler/include/Helper.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <iostream>
+#include <thread>
+#include <mutex>
+#include <condition_variable>
+#include <queue>
+#include <atomic>
+#include "Message.h"
+
+using namespace std;
+using std::lock_guard;
+using std::unique_lock;
+
+extern queue<Message> messages; // Shared queue to store messages;
+extern std::atomic<int> terminationCount;              // Atomic counter for termination count
+
+class Lockers {
+public:
+	static mutex _mtx_cout;          // Mutex for synchronization output
```

Comment:
why do you need this mutex ?, output is not a critical section!, only one consumer writes to it